### ISSUE TITLE
Add support for Ansible

### ISSUE THREAD

**Issue by arnabsinha4u** [inch-ci/inch_ci-web#166]
Is adding some stuff for Ansible?

**Comment by rrrene** [comment]
Interesting. What kind of "some stuff" do you have in mind?

**Comment by arnabsinha4u** [comment]
Example:
In Ansible we write playbooks. The playbooks have tasks and its a good practice to give a name to the tasks. We can live without naming the tasks as well but we can use the name to describe the task which behaves as documentation/issue report etc.

There are more ways to provide such details like usage of smarter tag naming, read lines.
Smarter implementations can be "Customer Journey" representations as well.

You can refer: https://github.com/arnabsinha4u/ansible-traininglab

**Comment by arnabsinha4u** [comment]
Do you have any updates of this ?

**Comment by rrrene** [comment]
Nope. I you want this functionality to exist, I am afraid you would have to first implement it yourself before we could think about adding it to Inch CI.

**Comment by dtoubelis** [comment]
There is no formal definition for embedding docs in Ansible as far as I know. Is this project contingent on that definition?
